iOS Example Frame/iOS Example Frame/MainViewController.swift

```diff
@@ -22,65 +22,100 @@ class MainViewController: UIViewController, CardViewControllerDelegate, ThreedsW
                               country: Country(iso3166Alpha2: "GB", dialingCode: "44"))
 
         let phone = Phone(number: "77 1234 1234",
                           country: Country(iso3166Alpha2: "GB", dialingCode: "44"))
-        let name = "User 1"
+        let name = "User Custom 1"
 
         let billingForm = BillingForm(name: name, address: address, phone: phone)
 
-        let viewController = CardViewController(checkoutAPIService: checkoutAPIService,
-                                                billingFormData: billingForm,
-                                                cardHolderNameState: .normal,
-                                                billingDetailsState: .required,
-                                                billingFormStyle: billingFormStyle,
-                                                defaultRegionCode: "GB")
+        cardViewController = createCardViewController(checkoutAPIService: checkoutAPIService,
+                                                      billingFormData: billingForm,
+                                                      billingFormStyle: billingFormCustom1Style)
+        cardViewController?.isNewUI = true
+        cardViewController?.availableSchemes = [.visa, .mastercard, .maestro]
+        pushCardViewController(cardViewController: cardViewController)
+    }
 
-        viewController.delegate = self
+    @IBAction func goToCustom2PaymentPage(_ sender: Any) {
+        let billingFormStyle = BillingFormCustom2Style()
+        let address = Address(addressLine1: "Test line Custom 2",
+                              addressLine2: nil,
+                              city: "London Custom 2",
+                              state: "London Custom 2",
+                              zip: "N12345",
+                              country: Country(iso3166Alpha2: "GB", dialingCode: "44"))
 
-        if let billingFormAddress = viewController.billingFormData?.address,
-           let billingFormPhone = viewController.billingFormData?.phone {
-            viewController.addressViewController.setFields(address: billingFormAddress, phone: billingFormPhone)
-        }
+        let phone = Phone(number: "77 1234 1234",
+                          country: Country(iso3166Alpha2: "GB", dialingCode: "44"))
+        let name = "User Custom 2"
 
-        return viewController
-    }()
+        let billingForm = BillingForm(name: name, address: address, phone: phone)
 
-    @IBAction func onClickGoToPaymentPage(_ sender: Any) {
-        navigationController?.pushViewController(cardViewController, animated: true)
+        cardViewController = createCardViewController(checkoutAPIService: checkoutAPIService,
+                                                      billingFormData: billingForm,
+                                                      billingFormStyle: billingFormStyle)
+        cardViewController?.isNewUI = true
+        cardViewController?.availableSchemes = [.visa, .mastercard, .maestro]
```

Comment:
Should this be part of the consumer configuration?

Reply:
This is in the main host project. (out of current scope) most of old code will be refactored later.